geom/dcel_debug.go

```diff
@@ -6,7 +6,7 @@ import (
 	"sort"
 )
 
-//nolint:deadcode,unused
+//nolint:unused
```

Comment:
`deadcode` was already disabled, so the `nolint` directive didn't need to include it.